deadcode.go

```diff
@@ -12,13 +12,25 @@ import (
 )
 
 var exitCode int
+var ignoreNames []string
+
+func init() {
+	flag.String("dir", ".", "directories to search, example: Foo1 Bar2")
+	flag.String("ignore", "", "sourcefile:name to ignore, example: somefile.go:aVariableFoo anotherfile.go:aConstBar")
```

Comment:
This may be a question of style, but I think these should go at the top of `main()`, before `flag.Parse()`. No reason to pull them out and initialize them separately, since they're only used there.